pipeline/src/datadownload.py

```diff
@@ -418,29 +418,52 @@ def convert_model_key_words(s, dictionary):
     )
 
 
-#Path to processed data directory
-gas_vehicles_csv = os.path.join(current_working_directory, 'data', 'processed', '1995_today_vehicle_fuel_consumption.csv')
-electric_vehicles_csv = os.path.join(current_working_directory, 'data', 'processed', 'Batteryelectric_vehicles__.csv')
-hybrid_vehicles_csv = os.path.join(current_working_directory, 'data', 'processed', 'Plugin_hybrid_electric_vehicles__.csv')
-
-#Creating a new directory for DuckDB tables
-database_directory = os.path.join(current_working_directory, 'data', 'database')
+# Path to processed data directory
+gas_vehicles_csv = os.path.join(
+    current_working_directory,
+    "data",
+    "processed",
+    "1995_today_vehicle_fuel_consumption.csv",
+)
+electric_vehicles_csv = os.path.join(
+    current_working_directory,
+    "data",
+    "processed",
+    "Batteryelectric_vehicles__.csv",  # noqa E501
+)
+hybrid_vehicles_csv = os.path.join(
+    current_working_directory,
+    "data",
+    "processed",
+    "Plugin_hybrid_electric_vehicles__.csv",
+)
+
+# Creating a new directory for DuckDB tables
+database_directory = os.path.join(
+    current_working_directory, "data", "database"
+)  # noqa E501
 Path(database_directory).mkdir(parents=True, exist_ok=True)
 
-#Creating DuckDB file at new directory
-duckdb_file_path = os.path.join(database_directory, 'car_data.duckdb')
+# Creating DuckDB file at new directory
+duckdb_file_path = os.path.join(database_directory, "car_data.duckdb")
```

Comment:
Shouldn't `"car_data.duckdb"` -> "car_data.duck.db"?

Reply:
Nevermind, lg

Reply:
You're right, this has been the file name format we've been working with.

I don't know why it still worked on my end, but I've changed the file name based on your comment.

Reply:
Nice, it worked (and I didn't know the reason before too) because DuckDB has its storage format (.duckdb). More info [here](https://www.christophenicault.com/post/large_dataframe_arrow_duckdb/)